src/test/java/io/opentracing/contrib/global/TrivialWrappingTracer.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2017 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.global;
+
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import io.opentracing.propagation.Format;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A 'trivial' wrapping tracer that delegates all calls as-is.
+ */
+final class TrivialWrappingTracer implements Tracer {
+    private static final Logger LOGGER = Logger.getLogger(TrivialWrappingTracer.class.getName());
+    static final AtomicInteger SEQUENCER = new AtomicInteger(0);
+
+    static final GlobalTracer.UpdateFunction WRAP = new GlobalTracer.UpdateFunction() {
+        @Override
+        public Tracer apply(Tracer current) {
+            try {
+                Tracer newWrapper = new TrivialWrappingTracer(current);
+                Thread.sleep(100);
+                return newWrapper;
+            } catch (InterruptedException e) {
+                throw new RuntimeException(e.getMessage(), e);
+            }
+        }
+    };
```

Comment:
IMO, I prefer a pessimistic lock mechanism to prevent this. After all, `update` is a low frequency operation. No need to race. This is easier and safer, I suppose. Thoughts?